### PR TITLE
Interpreter context missing files handling

### PR DESCRIPTION
**Pull request by ovstinga**

### CODE REVIEW

File: sdk/scenario-format/src/interpret_trait.rs [hunk 1]
```diff
@@ -19,6 +20,13 @@ impl InterpreterContext {
             ..self
         }
     }
+
+    pub fn with_missing_files(self, allow_missing_files: bool) -> Self {
```
Comment (andrei-marinica) [inline sdk/scenario-format/src/interpret_trait.rs:24]: Let's call this allow_missing_files, just to be clear, and no argument.

**Review by andrei-marinica — CHANGES_REQUESTED** [review]

**Review by CostinCarabas — APPROVED** [review]

File: sdk/scenario-format/src/interpret_trait.rs (hunk 1 above)
Comment (ovstinga) [reply sdk/scenario-format/src/interpret_trait.rs]: Done

File: sdk/scenario-format/src/interpret_trait.rs [hunk 2]
```diff
@@ -19,6 +20,13 @@ impl InterpreterContext {
             ..self
         }
     }
+
+    pub fn with_allow_missing_files(self) -> Self {
```
Comment (andrei-marinica) [inline sdk/scenario-format/src/interpret_trait.rs:24]: Sounds a bit awkward, `allow_missing_files()` or `with_allowed_missing_files()` maybe sounds better?

**Review by andrei-marinica — APPROVED** [review]

File: sdk/scenario-format/src/interpret_trait.rs (hunk 2 above)
Comment (ovstinga) [reply sdk/scenario-format/src/interpret_trait.rs:24]: Agree

**Pull request merged**